### PR TITLE
remove excessive comments

### PR DESCRIPTION
**Pull request by frankisawesome**
Signed-off-by: Frank Li <b439988l@gmail.com>

## PR description
Remove some unnecessary comments explaining conditional statements in a already very lengthy file

### CODE REVIEW

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 1]
```diff
@@ -2624,12 +2601,6 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
       }
 
       if (bootNodes == null) {
-        // We default to an empty bootnodes list if the option is not provided on CLI
-        // because
-        // mainnet bootnodes won't work as the default value for a custom genesis,
-        // so it's better to have an empty list as default value that forces to create a
-        // custom one
-        // than a useless one that may make user think that it can work when it can't.
```
Comment (macfarla) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:2632]: agree with deleting this one

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 2]
```diff
@@ -2595,17 +2583,6 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
...
-        // from the
-        // genesis file.
-        // We do the genesis parsing only in this case as we already have network id
-        // constants
-        // for known networks to speed up the process.
-        // Also we have to parse the genesis as we don't already have a parsed version
-        // at this
-        // stage.
-        // If no chain id is found in the genesis as it's an optional, we use mainnet
-        // network id.
```
Comment (macfarla) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:2608]: If no chain id is found in the genesis, use mainnet network id.

Comment (macfarla) [reply besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:2608]: ^ maybe a short comment is still useful here?

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 3]
```diff
@@ -2572,20 +2572,8 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
     final EthNetworkConfig.Builder builder =
         new EthNetworkConfig.Builder(EthNetworkConfig.getNetworkConfig(network));
 
-    // custom genesis file use comes with specific default values for the genesis
-    // file itself
-    // but also for the network id and the bootnodes list.
     if (genesisFile != null) {
-
-      // noinspection ConstantConditions network is not always null but injected by
```
Comment (macfarla) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:2580]: is this noinspection comment actually read by errorprone?

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 4]
```diff
@@ -2572,20 +2572,8 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
     final EthNetworkConfig.Builder builder =
         new EthNetworkConfig.Builder(EthNetworkConfig.getNetworkConfig(network));
 
-    // custom genesis file use comes with specific default values for the genesis
-    // file itself
-    // but also for the network id and the bootnodes list.
```
Comment (macfarla) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:2577]: agree with deleting this one

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java [hunk 5]
```diff
@@ -2572,20 +2572,8 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
...
-
-      // noinspection ConstantConditions network is not always null but injected by
-      // PicoCLI if used
       if (commandLine.getParseResult().hasMatchedOption("network")) {
-        // We check if network option was really provided by user and not only looking
-        // at the
-        // default value.
-        // if user provided it and provided the genesis file option at the same time, it
-        // raises a
-        // conflict error
```
Comment (macfarla) [inline besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:2588]: agree with deleting this one

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java (hunk 3 above)
Comment (frankisawesome) [reply besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:2580]: I don't think so - my IDE doesn't flag the following line after removing it

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java (hunk 2 above)
Comment (frankisawesome) [reply besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:2608]: 53bde8a

**Review by macfarla — APPROVED** [review]
LGTM

File: besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java (hunk 3 above)
Comment (macfarla) [reply besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java:2580]: ok

**Pull request merged**